programmers/claw-machine-game.js

```diff
@@ -0,0 +1,29 @@
+function solution(board, moves) {
+    const numbers = [];
```

Comment:
변수 선언도 안해주시고 변수를 쓰시면 동작이 안됩니다.
stackings는 무슨 의미인가요?
```suggestion
    const numbers = [];
    const stackings = [];
    let result = 0;
```

Reply:
인형이 뽑혀서 모이는 곳을 stackings라고 선언했습니다.  선언도 신경쓰고 좀 더 직관적인 변수명을 사용하겠습니다.